Formula/ppsspp.rb

```diff
@@ -50,10 +41,17 @@ def install
         prefix.install "PPSSPPSDL.app"
         bin.write_exec_script "#{prefix}/PPSSPPSDL.app/Contents/MacOS/PPSSPPSDL"
         mv "#{bin}/PPSSPPSDL", "#{bin}/ppsspp"
+        # FIXME: Currently bundled libMoltenVK.dylib is not compatible
+        # with Apple Silicon. Remove this when upstreamed.
+        Hardware::CPU.arm? && (rm "#{prefix}/PPSSPPSDL.app/Contents/Frameworks/libMoltenVK.dylib")
```

Comment:
```suggestion
        (prefix/"PPSSPPSDL.app/Contents/Frameworks/libMoltenVK.dylib").unlink
```
We should just remove this unconditionally, then, as we don't want to ship things that we didn't build from source. See https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae.

Reply:
PPSSPP will not build otherwise.

Reply:
I'm confused. You just said this above:

>PPSSPP will run just fine if I entirely remove libMoltenVK.dylib